scripts/setup_linux.sh

```diff
@@ -207,6 +207,9 @@ conda activate garage
   # Prevent pip from complaining about available upgrades
   pip install --upgrade pip
 
+  # 'Install' garage as an editable package
+  pip install -e .
```

Comment:
Please remove all things related to PYTHONPATH and PYTHON_ENV_VAR here and in setup_linux since we're using pip install now.

Reply:
Done